docs/get-started/python.md

```diff
@@ -0,0 +1,127 @@
+```console
+cdk8s init python-app
+```
+
+2. In your IDE, open the `main.py` file. 
+```python
+#!/usr/bin/env python
+from constructs import Construct
+from cdk8s import App, Chart
+
+
+class MyChart(Chart):
+   def __init__(self, scope: Construct, id: str):
+       super().__init__(scope, id)
+
+       # define resources here
+
+
+app = App()
+MyChart(app, "python")
+
+app.synth()
+```
+
+This sample shows the basic structure of a CDK8s application with the essential libraries: **constructs** and **cdk8s**. These libraries supply the foundational classes and methods required for working with CDK8s. It includes the following components:
+
+   * A custom `MyChart` class inherits from the [Chart](https://cdk8s.io/docs/latest/reference/cdk8s/python/#chart) base class provided by the cdk8s library, serving as a representation of the Kubernetes resources to be generated and managed.
+   * The `__init__` method within the `MyChart` class is responsible for initializing the base class ([Chart](https://cdk8s.io/docs/latest/reference/cdk8s/python/#chart)) and specifying the Kubernetes resources. This method is invoked when creating an instance of the class.
+   * An instance of the [App](https://cdk8s.io/docs/latest/reference/cdk8s/python/#app) class signifies the primary entry point of the CDK8s application and oversees the application's lifecycle and resources.
+   * An instance of the `MyChart` class is created by passing the app instance and a string identifier, "python", as arguments. This action generates and registers the Kubernetes resources defined in the `MyChart` class within the application. Note that in this example, the "python" string identifier is autogenerated based on the current directory name.
+   * The `synth` method is called on the `app` instance, which produces the required Kubernetes YAML manifest files based on the defined resources. Note that in this example, we haven't defined any resources within the `MyChart` constructor, so running the "cdk8s synth" command in the CLI would generate a blank Kubernetes manifest.
+
+## Defining Kubernetes resources
+Now, we are ready to define Kubernetes resources for our application. In this sample, we illustrate the creation of a basic Kubernetes [Service](https://kubernetes.io/docs/concepts/services-networking/service/) for a sample application. We start by importing the imports directory and the **k8s** sub-directory, which contains an `__init__.py` file where all CDK8s Kubernetes classes and functions are defined.
+
+### Copy the code sample
+1. Copy and paste the following code sample into the existing `main.py` file of your project.
+```python
+#!/usr/bin/env python
+from cdk8s import App, Chart
+from constructs import Construct
+from imports import k8s
+
+
+class AwesomeApp(Chart):
+   def __init__(self, scope: Construct, ns: str, app_label: str):
+       super().__init__(scope, ns)
+
+       # Define a Kubernetes Service
+       k8s.KubeService(self, "my-service",
```

Comment:
A while back we made the `k8s` classes available inside the cdk8s+ package - but didn't change the docs to use it. Now you do:

```python
import cdk8s_plus_26 as kplus

kplus.k8s.KubeService(...)
```

This is the recommended way now actually - maybe we can adjust? This would make it so we don't have to get into the weeds of the `imports` directory and how it is created

Reply:
We're not referencing cdk8s+ in this particular tutorial, but let me know if there's some other pythonic logic you wanted to interject.